dashboard/dashboard-front/src/components/admin/AdministrationView.vue

```diff
@@ -142,9 +142,21 @@ function deletePlugin(name: string, version: string) {
     })
 }
 
-function downloadPlugin(name: string, version: string) {
-  window.location.href = `/api/v1/plugin/${name}/${version}/download`
-}
+// If the url has a port, we are assuming the port is lifecycle, in which case we don't need to prepend /lifecycle
+// If the url does not have a port, we need to prepend /lifecycle
+const getBaseUrl = () => {
+    const { hostname, port } = window.location;
+    if (port) {
+        return ``;
+    } else {
+        return '/lifecycle';
```

Comment:
To make it simpler and less error-prone, I think we could always use prefixed `/lifecycle` base url, which is always accessible, even on localhost.

Reply:
I tried that, it didn't work, hence the extra logic.
![image](https://github.com/user-attachments/assets/ce28cf11-9b60-4382-84b6-2641cf88fc91)

Reply:
I owe you an explanation and I told you the wrong thing at first.

7103 is a port of Dashboard. So this works: http://127.0.0.1:7103/dashboard
Lifecycle runs on port 7102 and this address also works http://127.0.0.1:7102/lifecycle/

When you make something on Dashboard UI, it goes through Dashboard's backend (which forwards it to Lifecycle):
Dashboard UI (web app) -> Dashboard API -> Lifecycle API
This is because browsers are pretty restrictive about CORS and it's best for the web app to make a request to the same place it came from.

Thus, your base URL should be `/dashboard`